spec/spec_helper.rb

```diff
@@ -6,10 +6,11 @@
 require 'codeclimate-test-reporter'
 CodeClimate::TestReporter.start
 
+require 'pry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.